packages/delivery-fetch/tests/delivery.test.ts

```diff
@@ -293,10 +298,117 @@ describe('Browser Delivery', () => {
     }
 
     const deliveryFactory = createFetchDeliveryFactory(fetch, clock, backgroundingListener)
-    const delivery = deliveryFactory('/test')
+    const delivery = deliveryFactory('/test', false)
 
     const { state } = await delivery.send(deliveryPayload)
 
     expect(state).toBe('failure-discard')
   })
+
+  it('omits the bugsnag integrity header when not in a secure context', async () => {
+    const deliveryPayload: TracePayload = {
+      body: {
+        resourceSpans: [{
+          resource: { attributes: [{ key: 'test-key', value: { stringValue: 'test-value' } }] },
+          scopeSpans: [{
+            spans: [{
+              name: 'test-span',
+              kind: 1,
+              spanId: 'test-span-id',
+              traceId: 'test-trace-id',
+              endTimeUnixNano: '56789',
+              startTimeUnixNano: '12345',
+              attributes: [{ key: 'test-span', value: { intValue: '12345' } }],
+              droppedAttributesCount: 0,
+              events: []
+            }]
+          }]
+        }]
+      },
+      headers: {
+        'Bugsnag-Api-Key': 'test-api-key',
+        'Content-Type': 'application/json',
+        'Bugsnag-Span-Sampling': '1:1'
+      }
+    }
+
+    window.isSecureContext = false
```

Comment:
does this remain `false` for the next test? we're using `beforeAll` rather than `beforeEach` to revert this back to true

Reply:
Ah yeah 👍 fixed